subsys/net/lib/dhcpv4/dhcpv4_server.c

```diff
@@ -379,6 +380,22 @@ static uint8_t *dhcpv4_encode_subnet_mask_option(uint8_t *buf, size_t *buflen,
 	return buf + DHCPV4_OPTIONS_SUBNET_MASK_SIZE;
 }
 
+static uint8_t *dhcpv4_encode_router_option(uint8_t *buf, size_t *buflen,
+						                    struct in_addr *router)
```

Comment:
The alignment is still off, note that Zephyr uses 8 spaces/tab.

Reply:
When using all tabs, it may not align completely. Can I use spaces for last tab?